scripts/go_install.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+# See LICENSE.txt for license information.
+
+set -o errexit
```

Comment:
I often use xtrace mode to get more precise debug information

Reply:
All bash scripts should be consistent now :)